actions/admin_actions.jsx

```diff
@@ -254,6 +254,20 @@ export async function elasticsearchPurgeIndexes(success, error) {
     }
 }
 
+export async function blevePurgeIndexes(success, error) {
+    const purgeBleveIndexes = bindClientFunc({
```

Comment:
Are we calling here the client function instead of in redux because of the deprecation?

Reply:
It was initially put in the redux PR and moved here [out of a discussion there](https://github.com/mattermost/mattermost-redux/pull/1094#discussion_r397226988). The only piece needed in redux is the client function.